packages/arb-token-bridge-ui/src/components/TransferPanel/AdvancedSettings.tsx

```diff
@@ -91,8 +92,16 @@ export const AdvancedSettings = ({
   const { isEOA = false, isSmartContractWallet = false } = useAccountType()
 
   const [collapsed, setCollapsed] = useState(true)
+  const [inputLocked, setInputLocked] = useState(true)
   const [warning, setWarning] = useState<string | null>(null)
 
+  useEffect(() => {
```

Comment:
Because our `useAccountType` is async, we can't rely on it when setting a default `useState` value. This will always set it to `undefined` (or in our case `false` with `isEOA = false` in L92).

We need to update it once we get the result from `useAccountType`.